Remove Helm instructions from README (#124)
projectriff/charts: README.md

![](https://github.com/projectriff/charts/workflows/CI/badge.svg)

> # projectriff Installation YAML

Reviewer: ```suggestion
# projectriff Release YAML
```

> Installation YAML files for riff (includes Cert-Manager, Istio, Knative, Keda, KPack and Kafka dependencies).

Reviewer: ```suggestion
Release YAML files for riff (includes cert-manager, Istio, Knative, KEDA, kpack and Kafka dependencies).
```

Author: I have another issue for removing the riff charts #123

   ```sh
   riff_version=0.5.0-snapshot

>    ```

Reviewer: ```suggestion
```

>    ```sh

Reviewer: ```suggestion
```

make templates
```

> Package install YAML and charts locally. The install YAML will be placed in `uncharted` directory and charts will be placed in the `repository` directory.

Reviewer: ```suggestion
Package locally placing the YAML in the `uncharted` directory.
```
